charts/redpanda/values.yaml

```diff
@@ -470,40 +470,49 @@ storage:
       # -- Global flag that enables Tiered Storage if a license key is provided.
       # See the [property reference documentation](https://docs.redpanda.com/docs/reference/cluster-properties/#cloud_storage_enabled).
       cloud_storage_enabled: false
-      # -- See the [property reference documentation](https://docs.redpanda.com/docs/reference/tunable-properties/#cloud_storage_enable_remote_write).
+      # -- Cluster level default remote write configuration for new topics.
+      # See the [property reference documentation](https://docs.redpanda.com/docs/reference/tunable-properties/#cloud_storage_enable_remote_write).
       cloud_storage_enable_remote_write: true
-      # -- See the [property reference documentation](https://docs.redpanda.com/docs/reference/tunable-properties/#cloud_storage_enable_remote_read).
+      # -- Cluster level default remote read configuration for new topics.
+      # See the [property reference documentation](https://docs.redpanda.com/docs/reference/tunable-properties/#cloud_storage_enable_remote_read).
       cloud_storage_enable_remote_read: true
-
-      # -- Required for AWS and GCS.
+      # -- AWS or GCP region for where the bucket used for Tiered Storage is located (required for AWS and GCS).
       # See the [property reference documentation](https://docs.redpanda.com/docs/reference/cluster-properties/#cloud_storage_region).
       cloud_storage_region: ""
-      # -- Required for AWS and GCS.
+      # -- AWS or GCP bucket name used for Tiered Storage (required for AWS and GCS).
       # See the [property reference documentation](https://docs.redpanda.com/docs/reference/cluster-properties/#cloud_storage_bucket).
       cloud_storage_bucket: ""
-      # -- Required for AWS and GCS authentication with access keys.
+      # -- AWS or GCP access key (required for AWS and GCS authentication with access keys).
       # See the [property reference documentation](https://docs.redpanda.com/docs/reference/cluster-properties/#cloud_storage_access_key).
       cloud_storage_access_key: ""
-      # -- Required for AWS and GCS authentication with access keys.
+      # -- AWS or GCP secret key (required for AWS and GCS authentication with access keys).
       # See the [property reference documentation](https://docs.redpanda.com/docs/reference/cluster-properties/#cloud_storage_secret_key).
       cloud_storage_secret_key: ""
-      # -- See the [property reference documentation](https://docs.redpanda.com/docs/reference/cluster-properties/#cloud_storage_api_endpoint).
+      # -- AWS or GCP API endpoint.
+      # - For AWS, this can be left blank as it is generated automatically using the bucket and region (e.g. "<bucket>.s3.<region>.amazonaws.com")
```

Comment:
for AWS it ***should be*** left blank if the region and bucket name are set

Reply:
We have the same wording in our docs https://docs.redpanda.com/current/manage/tiered-storage/#:~:text=For%20AWS%2C%20this%20can%20be%20left%20blank.%20It%E2%80%99s%20generated%20automatically%20using%20the%20region%20and%20bucket

Could you file an issue in https://github.com/redpanda-data/docs/issues? 

Then we would change in both places.